### PR TITLE
Fix `clone` function in `ASTForStmt`

### LINKED ISSUE #742: `Clone` function in `ASTForStmt` isn't implemented properly
The class `ASTForStmt`  has two non-cloneable fields ((`ìnt`, `float`) and `str` don't implement the clone function). Given then that the nestml file has a `for-loop`, cloning the `AstNeuron` (in general the `ASTNode`) causes an exception to be thrown.

### PR DESCRIPTION
**Pull request by med-ayssar**
Fixes #742

### CODE REVIEW

File: pynestml/meta_model/ast_for_stmt.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def clone(self):
         """
         variable_dup = None
         if self.variable:
-            variable_dup = self.variable.clone()
+            variable_dup = str(self.variable)
```
Comment (clinssen) [inline pynestml/meta_model/ast_for_stmt.py:72]: Can we just assign ``variable_dup = self.variable``? Strings are immutable in Python. Same goes for the float and int below.

**Comment by med-ayssar** [comment]
@clinssen, aside note, can you check the pipeline, it is failing installing some dependencies.

File: pynestml/meta_model/ast_for_stmt.py (hunk 1 above)
Comment (med-ayssar) [reply pynestml/meta_model/ast_for_stmt.py]: Sure!

**Review by clinssen — APPROVED** [review]
Many thanks!

**Pull request merged**